data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -14,9 +14,10 @@ def __init__(self, df: pd.DataFrame) -> None:
         self.ENVIRONMENTAL_BURDEN_THRESHOLD: float = 0.90
         self.MEDIAN_HOUSE_VALUE_THRESHOLD: float = 0.90
         self.LACK_OF_HIGH_SCHOOL_MINIMUM_THRESHOLD: float = 0.10
+        self.FPL_200_SERIES: pd.Series = self.create_low_income_threshold(df)
```

Comment:
Hmm, yeah I'd prefer not to do this as part of the init. Mostly that's just setting properties and not doing a lot of  "work". Could you move to the beginning of `add_columns`? Also I think you'll need to update to point to `self._create_low_income_threshold` now that you've renamed it.

Reply:
Thanks for catching that.

so as I understand this, in `add_columns`:

`self.df[field_names.FPL_200_SERIES] = self._create_low_income_threshold(self.df)`

If this is the case, what would you suggest for naming `field_names.FPL_200_SERIES`?

Reply:
Yeah that sounds fine! `field_names.FPL_200_SERIES` could be something like, "Is low income". Wdyt?

Reply:
I'm down